stm32-modules/tempdeck-gen3/firmware/host_comms_task/usbd_desc.c

```diff
@@ -0,0 +1,281 @@
+/**
+  ******************************************************************************
+  * @file    usbd_desc_template.c
+  * @author  MCD Application Team
+  * @brief   This file provides the USBD descriptors and string formatting method.
+  *          This template should be copied to the user folder,
+  *          renamed and customized following user needs.
+  ******************************************************************************
+  * @attention
+  *
+  * <h2><center>&copy; Copyright (c) 2015 STMicroelectronics.
+  * All rights reserved.</center></h2>
+  *
+  * This software component is licensed by ST under Ultimate Liberty license
+  * SLA0044, the "License"; You may not use this file except in compliance with
+  * the License. You may obtain a copy of the License at:
+  *                      www.st.com/SLA0044
+  *
+  ******************************************************************************
+  */
+
+/* Includes ------------------------------------------------------------------*/
+#include "usbd_core.h"
+#include "usbd_desc.h"
+#include "usbd_conf.h"
+
+/* Private typedef -----------------------------------------------------------*/
+/* Private define ------------------------------------------------------------*/
+// TODO(frank 05-06-2022): Uncomment these VID/PID definitions and replace
```

Comment:
probably want to change these to something vaguely like the tempdeck - might forget otherwise.